.github/workflows/redfish-api-test.yml

```diff
@@ -0,0 +1,106 @@
+name: Redfish API Integration Tests
+
+on:
+  push:
+    branches: [ main, develop ]
+  pull_request:
+    branches: [ main, develop ]
+
+jobs:
+  redfish-newman-tests:
+    name: Redfish Newman/Postman Tests
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@0080882f6c36860b6ba35c610c98ce87d4e2f26f # v2.10.2
+        with:
+          egress-policy: audit
+
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+
+      - name: Set up Go
+        uses: actions/setup-go@3041bf56c941b39c61721a86cd11f3bb1338122a
+        with:
+          go-version: '1.24.x'
+
+      - name: Install dependencies
+        run: go mod download
+
+      - name: Install Newman
+        run: npm install -g newman newman-reporter-htmlextra
+
+      - name: Create minimal config file
+        run: |
```

Comment:
Instead of creating a new config file, can we not reuse the one available here:  https://github.com/nmgaston/console/blob/main/config/config.yml and modify only the needed configuration fields.